Makefile

```diff
@@ -113,7 +122,11 @@ integration-test-container:
 integration-test-deploy:
 	sudo docker push $(STORK_TEST_IMG)
 
-stork:
+codegen:
+	@echo "Generating CRD"
+	@hack/update-codegen.sh
+
+stork: codegen
```

Comment:
Don't need to gen code everytime when building stork

Reply:
Removed